ariadne/asgi.py

```diff
@@ -389,7 +392,12 @@ async def start_websocket_subscription(
             )
         else:
             results = cast(AsyncGenerator, results)
-            subscriptions[operation_id] = results
+            subscription = Subscription(
```

Comment:
Do we need a temporary value here? We could do `subscriptions[operation_id] = Subscription(...)`

Reply:
Yeah, I guess not, I changed that.